go/client/wallet_api_listen_display.go

```diff
@@ -0,0 +1,66 @@
+package client
+
+import (
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/stellar1"
+	"golang.org/x/net/context"
+)
+
+type walletNotificationDisplay struct {
+	*baseNotificationDisplay
+	cli stellar1.LocalClient
+}
+
+func newWalletNotificationDisplay(g *libkb.GlobalContext) *walletNotificationDisplay {
+	cli, err := GetWalletClient(g)
+	if err != nil {
+		panic(err.Error())
+	}
+	return &walletNotificationDisplay{
+		baseNotificationDisplay: newBaseNotificationDisplay(g),
+		cli: cli,
+	}
+}
+
+func (d *walletNotificationDisplay) displayPaymentDetails(ctx context.Context, accountID stellar1.AccountID, paymentID stellar1.PaymentID) error {
+	details, err := d.cli.GetPaymentDetailsLocal(ctx, stellar1.GetPaymentDetailsLocalArg{
+		AccountID: &accountID,
+		Id:        paymentID,
+	})
+	if err != nil {
+		return err
```

Comment:
we probably want to get this error out to the user instead of just returning it here

Reply:
probably we could print notifications in similar way to how message notifications are printed - where the returned object is of form `{ msg, error }`. It's not ideal but implementers haven't been complaining

Reply:
ah great point